pipenv/routines/update.py

```diff
@@ -110,6 +114,7 @@ def upgrade(
     categories=None,
     dev=False,
     lock_only=False,
+    extra_pip_args=None,
```

Comment:
How is extra_pip_args passed here? Is it a dict or a string? 

Can you add a test for that? Or at least a comment?

Reply:
I believe it is a string, I'll revisit soon.